doc/reduce-memory.md

```diff
@@ -0,0 +1,49 @@
+# Reduce Memory
+
+There are a few parameters that can be dialed down to reduce the memory usage of `bitcoind`. This can be useful on embedded systems or small VPSes.
+
+## In-memory caches
+
+The size of some in-memory caches can be reduced. As caches trade off memory usage for performance, reducing these will usually have a negative effect on performance.
+
+- `-dbcache=<n>` - the UTXO database cache size, this defaults to `450`. The unit is MiB (1024).
```

Comment:
```md
defaults to `450 MiB` (1 MiB = 1024 bytes).
```

Reply:
Thanks. However given the 4 ACKs I'm going to merge this.